packages/ledger/src/lib/ledger.ts

```diff
@@ -100,6 +100,7 @@ const Ledger: WalletBehaviourFactory<HardwareWallet> = async ({
   const getAccounts = (): Array<Account> => {
     return _state.accounts.map((x) => ({
       accountId: x.accountId,
+      publicKey: x.publicKey,
```

Comment:
The value of `publicKey` for ledger is without the `ed25519` we probably need to add it to keep it consistent with other wallets.